### PR TITLE
add event_type_id column

### PR DESCRIPTION
**Pull request by dpensi**
# Description

add event_type_id column

Fixes CCXDEV-8774

## Type of change

Please delete options that are not relevant.

- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Testing steps

- n/a

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: differ/storage.go [hunk 1]
```diff
@@ -547,10 +549,10 @@ func (storage DBStorage) ReadLastNotifiedRecordForClusterList(clusterEntries []t
 		clusterIDs[idx] = string(entry.ClusterName)
 	}
 
-	whereClause := fmt.Sprintf(` WHERE org_id IN (%v) AND cluster IN (%v) AND state = 1 `,
+	whereClause := fmt.Sprintf(` WHERE org_id IN (%v) AND cluster IN (%v) AND event_type_id = 1 AND state = 1`,
```
Comment (tisnik) [inline differ/storage.go:552]: very probably it will be faster to switch the WHERE conditions and put event_type and state on 1st place

**Review by tisnik — COMMENTED** [review]
commented

File: differ/storage.go [hunk 2]
```diff
@@ -481,7 +482,7 @@ func (storage DBStorage) WriteNotificationRecord(
 		notificationRecord.AccountNumber, notificationRecord.ClusterName,
 		notificationRecord.NotificationTypeID, notificationRecord.StateID,
 		notificationRecord.Report, notificationRecord.UpdatedAt,
-		notificationRecord.NotifiedAt, notificationRecord.ErrorLog)
+		notificationRecord.NotifiedAt, notificationRecord.ErrorLog, 1)
```
Comment (juandspy) [inline differ/storage.go:485]: maybe we should use a type instead of a hardcoded int? f.e `NotificationBackendEventTypeID`

Comment (dpensi) [reply differ/storage.go:485]: I'll use a constant

**Review by epapbak — APPROVED** [review]
+1

**Review by juandspy — APPROVED** [review]

**Pull request merged**